private_set_intersection/python/__init__.py

```diff
@@ -1,10 +1,13 @@
 """Private Set Intersection protocol based on ECDH and Bloom
 Filters.
 """
-from private_set_intersection.python import _psi_bindings
+try:
+    from private_set_intersection.python import _openmined_psi as psi
+except ImportError:
+    import openmined_psi as psi
```

Comment:
Why are these two cases needed? Maybe add a comment here.

Reply:
yeah, the first one is used by Bazel, the second one by the setuptools version